### PR TITLE
Lesson 11

### PR DESCRIPTION
**Pull request by OmegaLord**

### CODE REVIEW

File: spec/models/railway_station_spec.rb [hunk 1]
```diff
@@ -1,5 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe RailwayStation, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'is valid with valid attributes' do
+    expect(create(:railway_station)).to be_valid
```
Comment (AZatsepa) [inline spec/models/railway_station_spec.rb]: `expect(build(:railway_station)).to be_valid` - you cannot create invalid record

File: spec/models/route_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Route, type: :model do
+  it 'is valid with valid attributes' do
+    expect(create(:route)).to be_valid
```
Comment (AZatsepa) [inline spec/models/route_spec.rb]: The same

File: spec/models/train_spec.rb [hunk 3]
```diff
@@ -1,5 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe Train, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'is valid with valid attributes' do
+    expect(create(:train)).to be_valid
```
Comment (AZatsepa) [inline spec/models/train_spec.rb]: The same

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -13,7 +13,10 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+require 'factory_bot_rails'
```
Comment (AZatsepa) [inline spec/spec_helper.rb]: Do we need this requirement?

Comment (OmegaLord) [reply spec/spec_helper.rb]: without this requirement `uninitialized` constant FactoryBot`

**Pull request merged**